scripts/apps/users/directives/UserPreferencesDirective.ts

```diff
@@ -44,7 +44,7 @@ export function UserPreferencesDirective(
             const body = angular.element('body');
 
             scope.activeNavigation = null;
-            scope.activeTheme = localStorage.getItem('theme');
+            scope.activeTheme = localStorage.getItem('theme') !== '' ? localStorage.getItem('theme') : 'light-ui';
```

Comment:
It's not ideal to read local storage twice. I'd be nicer to do it like this:

```
scope.activeTheme = localStorage.getItem('theme');

if (scope.activeTheme === '') {
    scope.activeTheme = 'light-ui';
}
```

It would also be better to use a constant instead of inlining `light-ui`.